src/components/App/index.jsx

```diff
@@ -1,83 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
 import jsPDF from "jspdf";
 import 'svg2pdf.js';
 import './App.css';
 import TarsiaGrid from '../TarsiaGrid';
 import Questions from '../QuestionAnswer'
 import PrintableSVGDiv from '../PrintableSVGDiv'
-
 import hexGrid from '../../data/hexGrid';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.id = props.id;
-    this.state = {
-      values: {}
-    };
+const App = () => {
+  const [values, setValues] = useState({})
```

Comment:
Introduced the `userState` react hook to manage the state of this component. Hooks are super powerful and replace the state and lifecycle methods that the class components used:
https://reactjs.org/docs/hooks-intro.html

I also use array deconstruction here: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

What you end up with is a variable that contains our state data and a setter for re-setting that state.